### PR TITLE
Feat import ssh key

### LINKED ISSUE #16: Import should be implemented for all resource types
```
terraform import "env0_configuration_variable.test1" b94f954c-0b06-4fb4-8cb5-ed5161c99480
env0_configuration_variable.test1: Importing from ID "b94f954c-0b06-4fb4-8cb5-ed5161c99480"...

Error: Not implemented
```

:(

### PR DESCRIPTION
**Pull request by eranelbaz**
PR 1/4 for https://github.com/env0/terraform-provider-env0/issues/16
Enable `terraform import env0_ssh_key` using name or uuid or resource

### CODE REVIEW

**Comment by eranelbaz** [comment]
[TF Provider - Add import ability to all existing resources](https://trello.com/c/dn6qDaJ0/3287-tf-provider-add-import-ability-to-all-existing-resources)

File: env0/data_sshkey.go [hunk 1]
```diff
@@ -30,44 +30,66 @@ func dataSshKey() *schema.Resource {
...
+	sshKeys, err := apiClient.SshKeys()
+	var sshKey client.SshKey
+	if err != nil {
+		return client.SshKey{}, diag.Errorf("Could not query ssh keys: %v", err)
+	}
+	for _, candidate := range sshKeys {
+		if candidate.Name == name {
+			sshKey = candidate
+		}
+	}
```
Comment (roni-frantchi) [inline env0/data_sshkey.go:71]: We should error if there's more than one match by name

File: env0/resource_sshkey.go [hunk 2]
```diff
@@ -83,16 +71,19 @@ func resourceSshKeyDelete(ctx context.Context, d *schema.ResourceData, meta inte
...
-
-	// return []*schema.ResourceData{d}, nil
+	id := d.Id()
+	var getErr diag.Diagnostics
+	_, uuidErr := uuid.Parse(id)
+	if uuidErr == nil {
+		_, getErr = getSshKeyById(id, meta)
+	} else {
+		var sshKey client.SshKey
+		sshKey, getErr = getSshKeyByName(id, meta)
```
Comment (roni-frantchi) [inline env0/resource_sshkey.go:81]: We should log wether we query by name or id

Comment (roni-frantchi) [reply env0/resource_sshkey.go]: https://github.com/env0/terraform-provider-env0/pull/40/commits/7a2a7d32dce0fe8f8505200d619926a53823be29

File: env0/data_sshkey.go (hunk 1 above)
Comment (roni-frantchi) [reply env0/data_sshkey.go:74]: https://github.com/env0/terraform-provider-env0/pull/40/commits/5c7c8e1ccfb06b6ffc63481c846347fbdbbdec6e

**Review by roni-frantchi — APPROVED** [review]

**Pull request merged**